integration-test/content-scripts/gpc.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Should we add this to privacy test pages? I guess other platforms could also use such a test.